core/src/main/java/com/spotify/metrics/jvm/CpuGaugeSet.java

```diff
@@ -0,0 +1,70 @@
+package com.spotify.metrics.jvm;
+
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.Metric;
+import com.spotify.metrics.core.MetricId;
+import com.spotify.metrics.core.SemanticMetricSet;
+
+import java.lang.management.ManagementFactory;
+import java.lang.management.OperatingSystemMXBean;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A set of gauges reporting JVM CPU usage statistics.
+ */
+public class CpuGaugeSet implements SemanticMetricSet {
+
+  private final OperatingSystemMXBean operatingSystemMXBean;
+
+  private CpuGaugeSet(OperatingSystemMXBean operatingSystemMXBean) {
+    this.operatingSystemMXBean = operatingSystemMXBean;
+  }
+
+  @Override
+  public Map<MetricId, Metric> getMetrics() {
+    if (!(operatingSystemMXBean instanceof com.sun.management.OperatingSystemMXBean)) {
+      return Collections.emptyMap();
+    }
+
+    final com.sun.management.OperatingSystemMXBean osMxBean = (com.sun.management.OperatingSystemMXBean) operatingSystemMXBean;
+
+    final Map<MetricId, Metric> gauges = new HashMap<MetricId, Metric>();
+    final MetricId cpu = MetricId.build().tagged("what", "jvm-cpu-stats");
+
+    gauges.put(cpu.tagged("load", "process"), new Gauge<Double>() {
+      @Override
+      public Double getValue() {
+        return osMxBean.getProcessCpuLoad();
+      }
+    });
+
+    gauges.put(cpu.tagged("load", "system"), new Gauge<Double>() {
+      @Override
+      public Double getValue() {
+        return osMxBean.getSystemCpuLoad();
+      }
+    });
+
+    gauges.put(cpu.tagged("load", "system-average"), new Gauge<Double>() {
+      @Override
+      public Double getValue() {
+        return osMxBean.getSystemLoadAverage();
+      }
+    });
+
+    gauges.put(cpu.tagged("cpu", "process-cpu-time"), new Gauge<Long>() {
+      @Override
+      public Long getValue() {
+        return osMxBean.getProcessCpuTime();
```

Comment:
Doc for this `Returns the CPU time used by the process on which the Java virtual machine is running in nanoseconds`.

Since this is a different unit from *load*, it should probably have a distinct **what**.
Also, how useful is this value?